dataverse-persistence/src/test/java/edu/harvard/iq/dataverse/common/MarkupCheckerTest.java

```diff
@@ -1,136 +1,116 @@
 package edu.harvard.iq.dataverse.common;
 
-import org.junit.After;
-import org.junit.AfterClass;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.Test;
 
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.jupiter.api.Assertions.*;
 
 /**
  * @author rmp553
  */
 public class MarkupCheckerTest {
 
-    public MarkupCheckerTest() {
+    @Test
+    public void sanitizeBasicHTML_script() {
```

Comment:
For all the cases you can use @ParametrizedTest, but this is also fine.

Reply:
Done, perfect place to try junit5 parameters